bundles/org.openhab.binding.powermax/src/main/resources/OH-INF/thing/zone.xml

```diff
@@ -21,6 +21,12 @@
 			<channel id="last_trip" typeId="last_trip"/>
 			<channel id="low_battery" typeId="system.low-battery"/>
 			<channel id="bypassed" typeId="bypassed"/>
+			<channel id="alarmed" typeId="alarmed"/>
+			<channel id="tamper_alarm" typeId="tamper_alarm"/>
+			<channel id="inactive" typeId="inactive"/>
+			<channel id="tampered" typeId="tampered"/>
+			<channel id="zone_last_message" typeId="zone_last_message"/>
+			<channel id="zone_last_message_time" typeId="zone_last_message_time"/>
```

Comment:
To be coherent with my suggestion: `<channel id="last_status_time" typeId="last_status_time"/>`